app/controllers/games_controller.rb

```diff
@@ -22,7 +22,7 @@ def update
 
   def show
     @game = Game.includes(:pieces).find(params[:id])
-    @piece = Piece.find(params[:id])
+    #@piece = Piece.find(params[:id])
   end
```

Comment:
I looked back through my notes and can't recall why this was here. If it breaks something later we will have to take another look.